.github/pull_request_template.md

```diff
@@ -12,7 +12,7 @@ Uncomment if you're publishing docs for a prerelease version of dbt (delete if n
 - [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
 - [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
 -->
-- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
```

Comment:
hey @mirnawong1 : i just tested this ^ link. it works, it's not a "dead link" :

<img width="977" alt="link" src="https://user-images.githubusercontent.com/107218380/228581893-19ef915b-d14f-43cd-b2bf-3838366b44ec.png">

am i missing something? ty!

Reply:
oh right! it's the commented out section 'about versioning' that's dead. updated it !